docs/api/create-list.md

```diff
@@ -1,66 +1,73 @@
 <!--[meta]
 section: api
-title: createList
+title: Create List
+order: 2
 [meta]-->
 
-# keystone.createList(name, options)
+# createList(name, options)
 
 ## Usage
 
 ```javascript
 keystone.createList('Post', {
-  /* ... */
+  /* ...config */
 });
 ```
 
-## API
+### Config
 
-## name
+| Option          | Type                                | Default                       | Description                                                    |
+| --------------- | ----------------------------------- | ----------------------------- | -------------------------------------------------------------- |
+| `fields`        | `Object`                            |                               | Defines the fields in a list.                                  |
+| `schemaDoc`     | `String`                            |                               | A description for the list. Used in the Admin UI.              |
+| `hooks`         | `Object`                            | `{}`                          | Specify hooks to execute functions after list actions.         |
+| `labelField`    | `String`                            | `name`                        | Specify a field to use as a label for individual list items.   |
+| `labelResolver` | `Function`                          | Resolves `labelField` or `id` | Function to resolve labels for individual list items.          |
+| `access`        | `Function` \| `Object` \| `Boolean` | `true`                        | [Access control](/guides/access-control) options for the list. |
+| `adapterConfig` | `Object`                            |                               | Override the adapter config options for a specific list.       |
+| `itemQueryName` | `String`                            |                               | Changes the _item_ name in GraphQL queries and mutations.      |
+| `listQueryName` | `String`                            |                               | Changes the _list_ name in GraphQL queries and mutations.      |
+| `singular`      | `String`                            |                               | Specify a singular noun for Keystone to use for the list.      |
+| `plural`        | `String`                            |                               | Specify a plural for Keystone to use for the list.             |
+| `path`          | `String`                            |                               | Changes the path in the Admin UI.                              |
+| `plugins`       | `Array`                             | `[]`                          | An array of `plugins` that can modify the list config.         |
```

Comment:
A lot of these are missing their `default` value which makes it look like they're required.

Reply:
I put (required) when required. What would you suggest for no default? I originally thought `null` but... technically the default is not `null` I think nothing is best

Reply:
If the default is `undefined` would it make sense to specify how it will be resolved? 

schemaDoc for example: 
 `undefined` Will display in the docs as `A keystone list` 
adapterConfig:
`undefined` The `defaultAdapter`  specified when creating the Keystone instance will be used.
singular
`undefined` pluralize npm package will generate a singular label.

Or something to that note?

Reply:
Yeah I think that makes sense in a lot of cases

Reply:
I'm going to come back and revisit this later.